### PR TITLE
[Model] support bitsandbytes quantization with minicpm3 model

### PR DESCRIPTION
**Pull request by zixuanzhang226**

### CODE REVIEW

**Comment by DarkLight1337** [comment]
Can you link to an example HF repo that uses this quantization?

File: vllm/model_executor/models/minicpm3.py [hunk 1]
```diff
@@ -241,6 +241,23 @@ class MiniCPM3ForCausalLM(MiniCPMForCausalLM):
...
+    default_bitsandbytes_target_modules = [
+        ".q_a_proj.",
+        ".q_b_proj.",
+        ".kv_a_proj_with_mqa.",
+        ".kv_b_proj.",
+        ".o_proj.",
+        ".gate_proj.",
+        ".up_proj.",
+        ".down_proj.",
+    ]
```
Comment (Isotr0py) [inline vllm/model_executor/models/minicpm3.py:254]:
```suggestion

```
We don't need `default_bitsandbytes_target_modules` anymore once #10597 merged

Comment (zixuanzhang226) [reply vllm/model_executor/models/minicpm3.py]: I updated~ Please take a look. Thank you!

**Comment by zixuanzhang226** [comment]
> Can you link to an example HF repo that uses this quantization?
Please refer to https://huggingface.co/openbmb/MiniCPM3-4B

**Comment by DarkLight1337** [comment]
> > Can you link to an example HF repo that uses this quantization?
> > Hello @DarkLight1337 Please refer to https://huggingface.co/openbmb/MiniCPM3-4B

Oh, I forgot that you can perform in-flight quantization with bitsandbytes. Never mind.

**Review by jeejeelee — APPROVED** [review]
Thanks for your contribution, I can generate the reasonable result locally (TP=1/2)

**Comment by zixuanzhang226** [comment]
Hello @jeejeelee, would you please merge the code? Thank you!

**Pull request merged**